### DISCUSSION TITLE
Web Server Menu Empty

### DISCUSSION THREAD

**Discussion by Zosu** [gtxaspec/wz_mini_hacks#626] [Q&A] [answered]
Hi,
I'm having an issue where the web server menu for my Wyze v3 camera is just blank. I have followed all the instructions laid out in the setup guide and the device firmware version is 4.36.9.139. After enabling the web server in the config and loading up mini hacks, I can go to the web menu but nothing populates. Any ideas on what could be going on? Thanks.

![Screenshot 2023-09-28 205845](https://github.com/gtxaspec/wz_mini_hacks/assets/56417732/ad12ed6d-1d4f-4325-82e0-075508a277d4)

**Comment by gtxaspec** [comment, marked as answer]
hi @Zosu, are you using the latest release? see https://github.com/gtxaspec/wz_mini_hacks/issues/623#issuecomment-1730863669

**Reply by Zosu** [reply]
Yep, I'm using the latest release. Thank you for linking that. I'll give those suggestions a go and see if it solves it.

**Comment by Zosu** [comment]
Yep, looks like adding 
`WEB_SERVER_OPTIONS="cam config car jpeg status"`
brought back the web interface. Thanks! Now time to go troubleshoot why ssh always gets refused...
